lib/Doctrine/ORM/Cache/QueryCacheKey.php

```diff
@@ -32,15 +32,10 @@ class QueryCacheKey extends CacheKey
      */
     public $timestampKey;
 
-    /**
-     * @param string $hash      Result cache id
```

Comment:
It's ok to remove these, but the variable should be self-explaining. `$hash` is not such case.